src/plugins/intel_npu/src/al/include/intel_npu/al/config/npuw.hpp

```diff
@@ -49,6 +49,7 @@ DEFINE_OPT(NPUW_DCOFF_SCALE, bool, false, npuw::partitioning::dcoff_with_scale,
 DEFINE_OPT(NPUW_FUNCALL_FOR_ALL, bool, false, npuw::partitioning::funcall_for_all, CompileTime);
 DEFINE_OPT(NPUW_PARALLEL_COMPILE, bool, false, npuw::parallel_compilation, CompileTime);
 DEFINE_OPT(NPUW_WEIGHTS_BANK, std::string, "", npuw::weights_bank, CompileTime);
+DEFINE_OPT(NPUW_CACHE_DIR, std::string, "", npuw::cache_dir, CompileTime);
```

Comment:
How hard it'd be to implement a normal `CACHE_DIR`?

Reply:
Not possible on CPU currently. If I pass CACHE_DIR here, OV will throw an exception because we have no NPU backend instantiated

Reply:
lets discuss offline, it seems the point is missed here.

Reply:
Should be ok for now